assets/section-main-product.css

```diff
@@ -823,12 +823,13 @@ a.product__text {
   border-color: rgba(var(--color-foreground), var(--popup-border-opacity));
   border-style: solid;
   border-width: var(--popup-border-width);
-  filter: drop-shadow(
-    var(--popup-shadow-horizontal-offset)
-    var(--popup-shadow-vertical-offset)
-    var(--popup-shadow-blur-radius)
-    rgba(var(--color-foreground), var(--popup-shadow-opacity))
-  );
+  box-shadow: var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
+}
+
+.product-popup-modal__content.focused {
+  box-shadow: 0 0 0 0.3rem rgb(var(--color-background)),
+  0 0 0.5rem 0.4rem rgba(var(--color-foreground), 0.3),
+  var(--popup-shadow-horizontal-offset) var(--popup-shadow-vertical-offset) var(--popup-shadow-blur-radius) rgba(var(--color-shadow), var(--popup-shadow-opacity));
```

Comment:
I'm okay with this. 
But we will also need to modify the `:focus-visible` focus state. It's not showing the right box-shadow when focused now. The 0.3rem "background" border is missing. I think.

Expectation: 
<img width="982" alt="Screen Shot 2022-01-25 at 12 49 23 PM" src="https://user-images.githubusercontent.com/26778637/151031352-c007f5ba-772d-4ca2-afbf-5137871e6a83.png">

New focus experience: 
<img width="1009" alt="Screen Shot 2022-01-25 at 12 49 49 PM" src="https://user-images.githubusercontent.com/26778637/151031412-43c430ef-1d17-4217-b9ab-b82b94c8799b.png">